pkg/storage/storage_test.go

```diff
@@ -321,8 +347,8 @@ var _ = Describe("In-memory storage", func() {
 		err := testutil.CollectAndCompare(pointsStored, strings.NewReader(`
 		# HELP metrics_server_storage_points [ALPHA] Number of metrics points stored.
 		# TYPE metrics_server_storage_points gauge
-		metrics_server_storage_points{type="node"} 3
-		metrics_server_storage_points{type="container"} 5
+		metrics_server_storage_points{type="node"} 6
+		metrics_server_storage_points{type="container"} 10
```

Comment:
I would expect those metrics should stay the same value, wrote a comment above why.